docs/containers/debug-common.md

```diff
@@ -76,9 +75,16 @@ Example `launch.json` configuration for debugging a Python application:
 
 ## .NET
 
-More information about debugging .NET applications within Docker containers can be found in [Debug .NET within Docker containers](/docs/containers/debug-netcore.md).
+We offer two ways of building & debugging your project within Docker containers:
 
-Example `launch.json` configuration for debugging a .NET application:
+- **[With .NET SDK](/docs/containers/debug-netcore.md#net-sdk-container-build-debug-without-dockerfile)**: This option offers a simplified container debugging experience without the need for a `Dockerfile`. If you are familiar with `MSBuild` or want a straightforward approach, this is the recommended choice.
+  > Note: This option is only available for .NET SDK 7 and above and uses the `dotnet publish` command to build the image.
```

Comment:
There's a specific syntax for Notes in VS Code docs. This doesn't follow it exactly. It might need to be separated by a blank line, indented to align with the previous paragraph, and use bold (**Note**:) but let's find other Notes in the docs and follow the pattern.

Reply:
sounds good